docs/diagrams/data-flow/data-flow-api-endpoint.puml

```diff
@@ -0,0 +1,153 @@
+'
+'  Copyright (c) 2021 Daimler TSS GmbH
+'
+'  This program and the accompanying materials are made available under the
+'  terms of the Apache License, Version 2.0 which is available at
+'  https://www.apache.org/licenses/LICENSE-2.0
+'
+'  SPDX-License-Identifier: Apache-2.0
+'
+'  Contributors:
+'      Daimler TSS GmbH - Initial Version
+'
+
+@startuml
+
+skinParam NoteBackgroundColor WhiteSmoke
+skinParam NoteFontColor Black
+skinParam ParticipantBackgroundColor WhiteSmoke
+skinParam ActorBackgroundColor WhiteSmoke
+skinParam AgentBackgroundColor White
+skinParam AgentBorderColor SkyBlue
+skinparam shadowing false
+
+!define ConsumerColor f8f2ff
+!define ProviderColor d9edff
+!define WarningColor Business
+!define LeadColor Technology
+
+autonumber
+
+box Consumer #ConsumerColor
+    participant CBE as "Backend Service"
+    participant CCP as "Control Plane" #Business
+    participant CDP as "Data Plane" #Business
+end box
+
+box Provider #ProviderColor
+    participant PDP as "Data Plane" #Business
+    participant PCP as "Control Plane" #Business
+    participant PBE as "Backend Service"
+end box
+
+title
+ **Access to REST API**
+ <sub>Control Plane and Date Plane are part of the EDC.</sub>
+ <sub>Precondition: contract 'access to api' iuncluding constraints (e.g. how often, for which purpose etc.) is signed</sub>
+
+ Case 1a: Each API call matches one TransferProcess (http push)
+ <sub>Provider is pushing data to the Consumer</sub>
+end title
+
+CBE ->> CCP : getAsset(query params, int destination)
```

Comment:
Do you think of something when you defined the destination to be an int?

Reply:
Ahhh this means internal 🙈
Maybe adding a dot would make it clearer, thats these are abbreviations: `int.` and `ext.`